### ISSUE TITLE
Alternate Agent support

### ISSUE THREAD

**Issue by architdate** [petio-team/petio#314]
**Is your feature request related to a problem? Please describe.**
Petio cannot identify any agent apart from TVDB at the moment. Support for something like Hama.TV plugin and thereby allows people to request shows that already exist. So having support for it would be pretty helpful!

**Describe the solution you'd like**
Support for well known plugins such as hama :)

**Additional context**
Source Code is available here: https://github.com/ZeroQI/Hama.bundle
This uses its own scanner (Absolute Series Scanner): https://github.com/ZeroQI/Absolute-Series-Scanner/blob/master/Scanners/Series/Absolute%20Series%20Scanner.py
The metadata for hama has the following fields: https://github.com/ZeroQI/Hama.bundle/wiki/Plex-Metadata-Fields-Available

Hope this helps 👍 
Looking forward to this!

**Comment by AshDyson** [comment]
Yes Hama is planned we do support a number of meta agents though not just tvdb but any plug-in agents we will have to build integrations to map matches.
